pkg/preparer/setup.go

```diff
@@ -151,22 +151,41 @@ func getTLSClient(certFile, keyFile, caFile string) (*http.Client, error) {
 
 // GetStore constructs a key-value store from the given configuration.
 func (c *PreparerConfig) GetStore() (kp.Store, error) {
-	token, err := loadToken(c.ConsulTokenPath)
+	opts, err := c.getOpts()
 	if err != nil {
 		return nil, err
 	}
+	return kp.NewConsulStore(opts), nil
+}
 
-	client, err := getTLSClient(c.CertFile, c.KeyFile, c.CAFile)
+func (c *PreparerConfig) getOpts() (kp.Options, error) {
+	client := &http.Client{}
```

Comment:
any preference between `http.DefaultClient` versus `&http.Client{}` ?

(they're currently the same thing so I'm not fussed, but I ask anyway in case someone really cares)